src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -199,7 +199,7 @@ private X509Certificate2 EnsurePrivateKey(X509Certificate certificate)
             {
                 // Protecting from X509Certificate2 derived classes.
                 X509Certificate2 certEx = MakeEx(certificate);
-                
+
```

Comment:
Since there are no technical edits in this file it would be ideal if it was no longer part of this change.

`git checkout dotnet/master -- src/System.Net.Security/src/System/Net/Security/SecureChannel.cs`, IIRC (assuming you have a remote named `dotnet` corresponding to dotnet/corefx)

Reply:
will revert, I did manual revert, didn't notice (`-w`) this